topics/epigenetics/tutorials/cut_and_run/tutorial.md

```diff
@@ -216,32 +216,13 @@ The hg38 version of the human genome contains [alternate loci](https://www.ncbi.
 are present both in the canonical chromosome and on its alternate loci. The reads that map to these regions would map twice. To be able to filter reads falling into
 repetitive regions but keep reads falling into regions present in alternate loci, we will map on the Canonical version of hg38 (only the chromosome with numbers, chrX, chrY, and chrM).
 
-> <comment-title>Dovetailing</comment-title>
-> We will allow dovetailing of read pairs with Bowtie2. This is because adapters are removed by Cutadapt only when at least 3 bases match the adapter sequence, so it is possible that after trimming a read can contain 1-2 bases of adapter and go beyond it's mate start site. This occurs especially for CUT&RUN because the read length is quite short. Bowtie thus discards reads such as:
-> ```
-> ---------------------Mate 1--------------------------------->
-> <---------------------Mate 2----------------------
-> ```
->
-> or
->
-> ```
-> ---------------------Mate 1--------------------------------->
-> <---------------------Mate 2---------------------------------
-> ```
->
-> This is what we call dovetailing and we want to consider this pair as a valid concordant alignment.
-{: .comment}
-
-
```

Comment:
I decided to remove this part because the tutorial is using TrimGalore and removes adapters even if they are a single base pair.

Reply:
yes thats fine